apps/emqx/src/emqx_persistent_message.erl

```diff
@@ -74,6 +74,16 @@ storage_backend(Path) ->
     ConfigTree = #{'_config_handler' := {Module, Function}} = emqx_config:get(Path),
     apply(Module, Function, [ConfigTree]).
 
+-ifdef(STORE_STATE_IN_DS).
+initialize_session_ds_state() ->
+    ok = emqx_persistent_session_ds_state:open_db(storage_backend([durable_storage, sessions])).
+%% ELSE ifdef(STORE_STATE_IN_DS).
```

Comment:
It's usually a nice idea to add comments to the long macros, but our automatic formatter makes them so ugly it's not really worth it.